### DISCUSSION TITLE
Avoidance strategies for Brouter offline

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#245] [Q&A] [answered]
Is it conceivable to introduce offline avoidance strategies from level 0 to 5 for Brouter same as for former K2?
I have noticed that the Brouter option avoid main roads is interpreted as a very high avoidance level, Brouter then avoids them almost completely. Sometimes a weaker avoidance would be more desirable.

**Comment by devemux86** [comment, marked as answer]
@DolomitesTom indeed, a more detailed control would be useful.

I will have to check again what is possible in the profiles language.

Also, some BRouter presets would be nice,
for users who only like to change 1-2 settings.

---

Reminder: we will soon have new options in the BRouter profiles.
Stay tuned as I test their integration with Cruiser!

**Reply by DolomitesTom** [reply]
will do

**Comment by Ciutnik** [comment]
@DolomitesTom 
> Is it conceivable to introduce offline avoidance strategies from level 0 to 5 for Brouter same as for former K2?

In one of the previous versions of the Cruiser we had something like this (as far as I remember on a 0-100 percentage scale). And the problem was the proper "scaling". Depending on the specific case, even 10% avoidance was able to completely exclude a given category of roads. In some other case, those 10% made no difference. I guess that's why it was replaced with a zero-one switch.

Edit: I wish you a speedy recovery!!! This is the basis...

**Reply by DolomitesTom** [reply]
Thank you. I'm eagerly anticipating the autumn tour after this happened in the garage 8 days before the 8-day motorbike annual holiday. Garage work can also be dangerous. Take care ;-)
